bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -4,6 +4,7 @@ import _ from 'lodash';
 import PropTypes from 'prop-types';
 import { withModalMounter } from '/imports/ui/components/modal/service';
 import EndMeetingConfirmationContainer from '/imports/ui/components/end-meeting-confirmation/container';
+import MeetingEndedComponent from '/imports/ui/components/meeting-ended/component';
```

Comment:
Can now remove `import logoutRouteHandler from '/imports/utils/logoutRouteHandler';`

Please also remove ` className={styles.rating}` from meeting-ended/component.jsx - seems to be obsolete

Reply:
The code has been updated.